README.md

```diff
@@ -9,6 +9,7 @@
 [![Twitter][twitter-badge]][twitter-link]
 [![Matrix][matrix-badge]][matrix-link]
 
+<img src="https://raw.githubusercontent.com/RIOT-OS/RIOT/master/doc/doxygen/src/riot-logo.svg" width="75%"><!--
```

Comment:
Could it be centered ? for the moment it's left aligned:

![image](https://user-images.githubusercontent.com/1375137/133417703-3cdd3e01-e927-42de-80ff-3bbd93d40992.png)

Maybe make it even smaller (like 50% width)

Reply:
I tried both the `<center>` tag and `align="middle"` but Markdown didn't seem to pick it up.

Reply:
50% looks a bit small IMO, but 66% works well

Reply:
IMO, centered it looks like too far right. I prefer non-centered.

Reply:
I agree with @kaspar030. The width of the parsed README depends very much on the user resolution and at some point it looks weird.

Reply:
> I agree with @kaspar030. The width of the parsed README depends very much on the user resolution and at some point it looks weird.

But let's not bikeshed this any further... ACK!